src/sentry/runner/commands/backup.py

```diff
@@ -217,9 +251,14 @@ def json_lines(obj: JSONData) -> list[str]:
             # don't mutate their inputs), this should be sufficient to ensure that the order in
             # which comparators are applied does not change the final output.
             for cmp in comparators[id.model]:
+                ex = cmp.existence(id, exp, act)
```

Comment:
noticed a typo in the comment above:
`This ensures tha, in cases where a single model uses multiple` -> `This ensures that, in cases where a single model uses multiple`